yunDoorbotoFirmware/yunDoorbotoFirware.ino

```diff
@@ -25,7 +26,12 @@ void setup(){
   pinMode(RED_LED, OUTPUT);   // use LED
   pinMode(GREEN_LED, OUTPUT); // use LED
   pinMode(RELAY, OUTPUT);     // make relay pin an output
-}
+  
+  //blink LED's during bootup
+  blink(RED_LED, 15, 200);
+  delay(OPEN_TIME); //delay for a short period
+  blink(GREEN_LED, 15, 200);
```

Comment:
This will prevent start up for 3400ms. In production that's not a big deal, but it will make it annoying to test with. Normally I would avoid delays like the plague in code. However, in this firmware it was advantageous to block potential brute forcing attempts at the reader effectively slowing down the number of request that could be made. I would say taking a <500ms to show something like this would be fine. I also have a library that emulates the set timeout function found in JS that would make it non-blocking https://github.com/PaulBeaudet/JS_Timer